src/powershell/Public/Remove-FinOpsCostExport.ps1

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+        Delete a Cost Management Export and optionally data associated with the export.
+
+    .PARAMETER Name
+        Name of the Cost Management Export.
+
+    .PARAMETER Scope
+        The scope associated with export operations. This includes '/subscriptions/{subscriptionId}/' for subscription scope,
+        '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope,
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope and '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope,
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope,
+        '/providers/Microsoft.Management/managementGroups/{managementGroupId} for Management Group scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billingProfile scope,
+        '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/invoiceSections/{invoiceSectionId}' for invoiceSection scope,
+        and '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/customers/{customerId}' specific for partners.
+
+    .PARAMETER RemoveData
+        Optional. Indicates that all cost data associated with the Export scope should be deleted.
+        This will delete all data in the storage account associated with the export scope (billing, subscription, management group, resource group).
+
+    .PARAMETER APIVersion
+        Optional. Specify an alternative API version. Default is 2023-04-01-preview.
+
+    .EXAMPLE
+        Remove-FinOpsCostExport -Name MyExport -Scope "/subscriptions/00000000-0000-0000-0000-000000000000" -RemoveData
+
+        Deletes a Cost Management Export named MyExport scoped to /subscriptions/00000000-0000-0000-0000-000000000000, and deletes all data associated with that scope.
+#>
+
+function Remove-FinOpsCostExport
+{
+  [CmdletBinding(SupportsShouldProcess)]
+  param
+  (
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Name,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Scope,
+
+    [Parameter()]
+    [switch]
+    $RemoveData,
+
+    [Parameter()]
+    [string]
+    $APIVersion = '2023-04-01-preview'
+  )
+
+  try
+  {
+    $additionalParamaters = @{
+      scope = $scope
+    }
+    $payload = ConvertTo-Json -InputObject $additionalParamaters -Depth 100
+
+    # TODO : Validate parameters and resources before proceeding
+    $httpResponse = Invoke-AzRestMethod `
```

Comment:
I'm not familiar with this command, but how is the scope applied here? I see it's in the payload, but I'm not sure how the command turns that into an actual HTTP request.

Reply:
It's a parameter in the Payload.

Reply:
https://github.com/Azure/azure-powershell/blob/main/src/Accounts/Accounts/help/Invoke-AzRestMethod.md

Reply:
I see there's a `-Payload` parameter, but there's nothing that talks about how it's used. The "magic" makes me a bit nervous. I'd prefer using `-Path` personally, since it's easier to spot issues. I just worry there's something that isn't supported by the magic that will end up failing on us. Maybe it's just me. As long as we can confirm it works with billing accounts and subscriptions, then I suppose we should be fine.